src/components/dev-dot-content/mdx-components/mdx-heading-permalink/mdx-heading-permalink.module.css

```diff
@@ -0,0 +1,33 @@
+.root {
+	composes: g-focus-ring-from-box-shadow from global;
+	display: inline-block;
+	opacity: 0;
+	transition: opacity 0.2s ease;
+	order: 1;
+	position: relative;
+	left: -22px;
+	margin-right: -20px;
+	border-radius: 5px;
+	padding: 2px;
+
+	@media (--dev-dot-sidecar-up) {
+		left: -34px;
+		margin-right: -26px;
+		padding: 5px;
+		top: auto;
+	}
+
+	& .icon {
+		height: 16px;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+	}
+
+	&:hover,
+	&:focus {
+		background: var(--token-color-palette-neutral-200);
+		opacity: 1;
+		transition: opacity 0.2s ease;
```

Comment:
Is this necessary given the above `transition` declaration?

Reply:
Nope! Good catch that was redundant. deleted here:
https://github.com/hashicorp/dev-portal/pull/1089/commits/e42832beb39aeacda8770d588b4713480f1233b4